src/script/app.js

```diff
@@ -0,0 +1,10 @@
+let scroll1 = window.pageYOffset;
+window.onscroll = function () {
+    let scroll2 = window.pageYOffset;
+    if (scroll1 > scroll2) {
+        document.querySelector('nav').style.top = "0";
+    } else {
+        document.querySelector('nav').style.top = "-80px";
```

Comment:
You use `top` property for this feature, which is not valid. By setting `top` you cause browser to do multiple more steps during rendering, which is not optimal. Ideally you would want to use `transform` property to introduce this kind of animation, because it does not cause recalculation of the layout by the browser.